docs/en/02_Creating_pages_and_content/00_Pages/05_Rolling_Back_Pages.md

```diff
@@ -20,6 +23,20 @@ When a page is rolled back to the previous version, it is only rolled back in th
 
 To see exactly what was changed between 2 different versions of the page:
 
-1. Navigate to the ***History*** tab, select the **Compare mode** checkbox field.
-2. Select 2 different snapshots from the left sidebar. To deselect a snapshot, select it again.
-3. The preview on the right will then highlight everything that has been removed from the page in red, and everything that was added in green.
+Navigate to the ***History*** tab, to enter **Compare mode** you can either:
+* Click the button ***Compare mode*** shown as a slider icon and select the ***Compare two versions*** checkbox field or
+* Select a record from the list and click the button ***Compare***.
+
+![Compare mode](/_images/compare_mode.png)
+
+Once you have entered compare mode select 2 different records.
+**Note:** Your selected record will automatically be positioned at the top of the page separated by a black bar. Your record will also appear in the below list with the label **Already selected**. Select another record to compare versions.
+
+![Selecting two versions](/_images/selecting_versions.png)
+
+The preview will then highlight everything that has been removed from the page in red, and everything that was added in green between those two versions in a read-only format.
+
+![Comparing two versions](/_images/comparing_versions.png)
```

Comment:
I'm not convinced about using leading slashes for the file path - you can see from the preview on GitHub that this is not working. We'd be better to use relative paths e.g. `../../_images/foo.png`. That said, this seems to be common across the other parts of this repo already, and it works on the website so it's probably OK for now =)